tests/test_config_import.py

```diff
@@ -91,3 +97,49 @@ def test_import_multi_input_field(seed_dynamic_data, _db):
     assert multi_input_component
     assert multi_input_component.type == ComponentType.MULTI_INPUT_FIELD
     assert len(multi_input_component.children) == 4
+
+
+def test_creates_unique_lists_per_form_with_shared_list_name(app: Flask, clean_db: SQLAlchemy):
```

Comment:
Can we have the delete functionality test as well ?

Reply:
Delete functionality hasn't been changed here and I reckon we probably want to scope that separately; as per comment above:

> Regarding template deletion and list deletion, this is a good question - maybe we can create a separate ticket for ensuring cascade delete across the data schema as an outcome from your excellent work on the [FAB code end-to-end understanding](https://mhclgdigital.atlassian.net/browse/FS-4900) ticket?

Assuming we do this, we can add tests as part of this work to ensure cascade delete works properly. Good thinking to test any new functionality though 👍